### PR TITLE
Make up for PR #564.

### PR DESCRIPTION
**Pull request by H1Gdev**
#### What does this implement/fix? Explain your changes.

- Make up for PR #564.
- Pass pservers-pattern-03-f.svg.

### CODE REVIEW

File: doc/ReleaseNotes.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ The release versions are NuGet releases.
 * fixed rendering of text-anchor `middle` (see [#769](https://github.com/vvvv/SVG/issues/769)
 * fixed opacity not correctly inherited issue (see [#777](https://github.com/vvvv/SVG/issues/777) and [PR #788](https://github.com/vvvv/SVG/pull/788))
 * fixed fallback color (see [#779](https://github.com/vvvv/SVG/issues/779) and [PR #792](https://github.com/vvvv/SVG/pull/792))
+* make up for [PR #564](https://github.com/vvvv/SVG/pull/564) (see [PR #793](https://github.com/vvvv/SVG/pull/793))
```
Comment (mrbean-bremen) [inline doc/ReleaseNotes.md:24]: better something like: "fixed regression introduced by..." (at least I wouldn't understand this one)

**Review by mrbean-bremen — COMMENTED** [review]
Thanks - does this fix #779?

**Comment by H1Gdev** [comment]
@mrbean-bremen 

No. This is fix for another issue.

**Comment by mrbean-bremen** [comment]
Ok, thanks!

**Comment by H1Gdev** [comment]
@mrbean-bremen 
There was a problem so I will fix this PR.
please wait a moment.

**Comment by mrbean-bremen** [comment]
Ok, too late...

**Comment by H1Gdev** [comment]
@mrbean-bremen 
I created PR #794.

**Pull request merged**